src/components/App/App.tsx

```diff
@@ -16,6 +17,7 @@ const App: React.FC<AppProps> = () => {
         <Route path='/' element={<Home />} />
         <Route path='/:year' element={<AlbumsByYear />} />
         <Route path='/:year/:album_id' element={<AlbumDetail />} />
+        <Route path='*' element={<ErrorPage />}/>
```

Comment:
I think you need a `/` before the `*` here. 
So it should be:
`<Route path='/*' element={<ErrorPage />} />`